torch/_inductor/lowering.py

```diff
@@ -545,7 +546,12 @@ def _to_dtype(x):
 
 @register_lowering(prims.convert_element_type, type_promotion_kind=None)
 def _convert_element_type(x: TensorBox, dtype: torch.dtype):
-    if dtype.is_complex or x.get_dtype().is_complex:
+    is_complex = dtype.is_complex or x.get_dtype().is_complex
+    is_float8 = "float8" in str(dtype) or "float8" in str(x.get_dtype())
```

Comment:
nit: maybe exact match instead of looking at string match in name?

Reply:
Can the logic be here instead ? https://github.com/pytorch/pytorch/blob/main/torch/_inductor/lowering.py#L1447

I don't even know why we have the complex check in this code, I would expect the other function to supersede it

Reply:
Used wrong link - but yea meant to link to reference fallback_node_due_to_unsupported_type as @peterbell10